src/tasks/onetime/AutoCombatLogic.py

```
@@ -70,6 +111,150 @@ def _periodic_search(self):
+                had_action —— 本帧是否产出了条件动作（供立即释放判断）。
+        """
+        # 战技重试：上一帧 digit token 因技力不足失败，本帧重试同一 token（上限 15 帧）
+        if self._pending_skill_token is not None:
+            token = self._pending_skill_token
+            self._pending_skill_frames += 1
+            if self._pending_skill_frames >= self._SKILL_RETRY_MAX_FRAMES:
+                # ponytail: 15 帧（≈1.5s）仍未恢复 → 跳过该 token，推进生成器
+                self.task.log_info(f"技力不足超时 {self._pending_skill_frames} 帧，跳过战技 {token}")
+                self._pending_skill_token = None
+                self._pending_skill_frames = 0
+                # 推进生成器（不返回 had_action，让立即释放有机会触发）
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)